python-sdk/tests/e2e/conftest.py

```
@@ -137,3 +139,28 @@ def create(**kwargs: Any) -> Any:
+
+    def _create(name: str) -> Any:
+        tag = langwatch.prompts.tags.create(name)
+        tracked_names.append(name)
+        return tag
+
+    yield _create
+
+    for name in tracked_names:
+        try:
+            langwatch.prompts.tags.delete(name)
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)